app/views/passengers/index.haml

```diff
@@ -1,30 +1,54 @@
+%p
+  %b.yay
```

Comment:
Two things:

1.  What does the class `yay` represent? What are we celebrating? I'm being a little sarcastic, but in truth this class name doesn't seem to be meaningful.
2.  Don't use `<b>` tags. ([generally](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong#Bold_vs._Strong)) Use `<strong>` if you actually want to emphasize the text over its surrounding text.  Otherwise, you can modify the `font-weight` on your class's css.